packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -236,3 +236,90 @@ export const styles = css`
         `
     )
 );
+
+export const primaryButtonStyles = css``.withBehaviors(
```

Comment:
I'm not sure if the anchor-button should have appearance variants. I think it would be a good discussion to have as a larger group to pull in the designers. My initial thought is that it probably makes sense to have the variants on the anchor-button.

My rationale is this:
- From a user's perspective (and possibly a designer's perspective), a button is something that you click on to take some action. That action might be to interact with the current page, or it might be to perform a navigation. With that reasoning, any place that you could have a button, but the action is associated with navigation, you'd want to use an anchor-button. Since the difference is somewhat subtle, you should be able to style the anchor-button in all the same ways as the button. I think this aligns with saying that an anchor within another component (such as the nimble-tree or the nimble-menu) should look identical as their non-anchor counter parts.
- The toggle-button is a completely different control though -- it's more of a different way to render a switch than it is a different way to render a button. So, unless specifically requested by a designer, I don't think it would inherently require all the appearance variants as a regular button.
- The menu-button is a bit foggy in my mind because I think there are cases where menu-buttons would be used alongside regular buttons (like in a toolbar) and you want it to be visualized the same way. However, I'm not sure it has the same "call-to-action" type styling that a button would have (which is my understanding of why we have those variants on a regular button). Things get further complicated because we implemented the menu-button with a toggle-button, so I think we would need to keep support for those two components in sync.

Again, just my opinions. I'd be interested in seeing what Brandon/Leslie's visions are for these components.

Reply:
I'll bring it up with Brandon/Leslie, but for the time being, I'll leave the prominent appearance variant on the anchor button.